src/plugins/discover/public/application/main/data_fetching/fetch_all.ts

```diff
@@ -98,6 +98,8 @@ export function fetchAll(
       sendLoadingMsg(dataSubjects.totalHits$, {
         result: dataSubjects.totalHits$.getValue().result,
       });
+    } else {
+      sendLoadingMsg(dataSubjects.totalHits$);
```

Comment:
Maybe we should keep the previous `result` too?

Reply:
I'm not sure about that, because I think if I send a `loading`, the previous result is not longer valid? So if we rely on a previous result somewhere we might change this

Reply:
You're both right, let's simplify the code ... done